Week-2/article/styles.css

```diff
@@ -0,0 +1,31 @@
+img {
+    width: 300px;
+}
+
+.first_image {
```

Comment:
any class/id/variable name should be meaningful. 

.logo would be a better class name in this case

Reply:
okay